### PR TITLE
closed #121

### PR DESCRIPTION
**Pull request by montogeek**

### CODE REVIEW

**Comment by Patineta** [comment]
@montogeek Thanks for fixing the issue!

File: css/app.css [hunk 1]
```diff
@@ -611,8 +611,12 @@ body.has-notification #notification {
 }
 .popcorn-load.hidden {
   opacity: 0;
+  z-index: 0;
   pointer-events: none;
 }
+.popcorn-load.hidden .spinner {
+  -webkit-animation: none;
```
Comment (samccone) [inline css/app.css:618]: this needs to be 

`@include animation(none);`

Comment (montogeek) [inline css/app.css:618]: That mixin doesn't exists.

Comment (samccone) [inline css/app.css:618]: Oh whoops! I was thinking we were using something like this https://github.com/ericam/compass-animation already.

nm then!

Comment (montogeek) [inline css/app.css:618]: Maybe in the future if the application has more animations would be a good idea to use it. Thanks :)

**Pull request merged**